lib/api-client.js

```diff
@@ -6,22 +6,21 @@ var apiClient = new JSONAPIClient(config.host + '/api', {
   'Accept': 'application/vnd.api+json; version=1',
 }, {
   beforeEveryRequest: function() {
+    console.log('using local panoptes-client');
     var auth = require('./auth');
     return auth.checkBearerToken();
   },
 
-  handleError: function(response) {
+  handleError: function(error) {
     var errorMessage;
-    if (response instanceof Error) {
-      throw response;
-    } else if (typeof response.body === 'object') {
-      if (response.body.error) {
-        errorMessage = response.body.error;
-        if (response.body.error_description) {
-          errorMessage += ' ' + response.error_description;
+    if (typeof error.response.body === 'object') {
+      if (error.response.body.error) {
```

Comment:
Errors should maybe follow the [JSON API spec for error objects](http://jsonapi.org/format/1.1/#error-objects).